front/src/modules/views/hooks/internal/useViewFilters.ts

```diff
@@ -21,7 +21,9 @@ export const useViewFilters = (viewScopeId: string) => {
   });
   const apolloClient = useApolloClient();
 
-  const { currentViewFiltersState } = useViewScopedStates();
+  const { currentViewFiltersState } = useViewScopedStates({
+    customViewScopeId: viewScopeId,
```

Comment:
why do we need to do this change?

Reply:
Because otherwise useView can't be called from outside of the viewScope I think.